### ISSUE TITLE
Running NSS locally Error on Registration

### ISSUE THREAD

**Issue by rodant** [solid/node-solid-server#1189]
I installed the solid server as described in the README:

npm install -g solid-server
and then
solid start

I'm using an slightly changed version of the default config.json just setting my certificate and key. The server starts and I can load the URL: https://localhost:8443 correctly, but when a click on "Register", I get

```
ENOENT: no such file or directory, open '/home/antonio/backup_ignore/spoter.me/docker-solid-server/config/views/layouts/main.hbs'
 ```
i'll appreciate any support for this issue.

**Comment by thibautRe** [comment]
Running into the same issue, I can provide a bit more debugging info:

- MacOS v10, Node v12.2
- I tried both `solid-server@5.0.1` and `solid-server@4.4.2`, the error was the same
- Any URL that isn't `/` will end up showing this error. So both `/register`, `/login` and `/blablabla` will show the message.
- There is no `layouts` folder in `config/views`.
- Creating both the folder `layouts` and an empty `main.hbs` in it will just show an empty page everywhere.

I couldn't find any reference to a `layouts` or `main.hbs` folder in the codebase, so that's pretty much where my debugging help stops.

**Comment by dmitrizagidulin** [comment]
@rodant I believe you need to run ‘solid init’ after install.

**Comment by rodant** [comment]
@dmitrizagidulin After running solid init the issue persists. I've also experimented with different node versions. 8.11.2, 10.0.0 and 11.6.0, in all cases the issue shows up. 

I could avoid the problem by using the docker setup of https://github.com/angelo-v/docker-solid-server which uses node 10 and the same NSS version 5.0.1, which is very strange for me, but I can find what is making the difference. Also building the docker image from this repo and running it produces the same issue again.

**Comment by rodant** [comment]
My config.json file looks like this:

```
{
  "root": "/home/antonio/backup_ignore/spoter.me/node-solid-server/data",
  "port": "8443",
  "serverUri": "https://solid.example:8443",
  "webid": true,
  "mount": "/",
  "configPath": "./config",
  "configFile": "./config.json",
  "dbPath": "./.db",
  "sslKey": "/home/antonio/tmp/solid/certs/solid.example.key",
  "sslCert": "/home/antonio/tmp/solid/certs/solid.example.crt",
  "multiuser": true,
  "enforceToc": true,
  "tocUri": "http://localhost",
  "disablePasswordChecks": true,
  "supportEmail": "",
  "server": {
    "name": "localhost",
    "description": "",
    "logo": ""
  }
}
```

**Comment by rohithate** [comment]
I'm also having a similar issue after migrating from 4.4.0 to 5.01. 

Sever is running in a multi-user environment. When I start server, console  shows that it's running solid server (5.0.1), although on accessing root uri, it show that it's Running on Solid 4.4.0. This does not go away even after re-installing solid server and running ‘solid init’

**Comment by jpnp** [comment]
I've had this issue on new installs.  It seems to be related to express-handlebars which moved from v3.0.2 to v3.1.0 last week.

I've found I can work round it by doing an `npm install "express-handlebars@3.0.2"` before `npm install solid` to ensure that it uses a 3.0 version of the dependency.

**Comment by ewingson** [comment]
perhaps it' s related here, after upgrading to 5.1.0 and installing express-handlebars still remains 'Error: ENOENT: no such file or directory, open '/var/www/html/config/views/layouts/main.hbs'`

**Comment by rodant** [comment]
I can confirm @jpnp, after removing version 3.1.0 from express-handlebars and install version 3.0.2, the server works correctly. I think the team should consider setting an exact match for the version of all dependencies, otherwise issues like this will potentially appear and involve a lot of time to clarify.

**Comment by rohithate** [comment]
> 
> 
> I've had this issue on new installs. It seems to be related to express-handlebars which moved from v3.0.2 to v3.1.0 last week.
> 
> I've found I can work round it by doing an `npm install "express-handlebars@3.0.2"` before `npm install solid` to ensure that it uses a 3.0 version of the dependency.

@jpnp thank you for suggesting the workaround and directing towards the origin of problem.👍 

I needed `solid-server` installed globally and installing `express-handlebars` before `solid-server` didn't help me. Though, I was able to solve it as follows:
- I downloaded the source code [v5.1.0](https://github.com/solid/node-solid-server/releases/tag/v5.1.0).
- extracted it and modified the package.json file from `"express-handlebars": "^3.0.0"` to `"express-handlebars": "3.0.0"`. 
- compressed it back and installed a `solid-server` from the new file. `npm install -g node-solid-server-5.1.0.tar.gz`
